includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -1006,6 +1012,82 @@ public function attach_currency_selector_element() {
+		}
+
+		$this->maybe_add_presentment_metadata_to_order( $order );
+
+		$presentment_amount   = (int) $order_helper->get_stripe_presentment_amount( $order );
+		$presentment_currency = $order_helper->get_stripe_presentment_currency( $order );
+
+		if ( ! $presentment_amount || ! $presentment_currency ) {
+			return;
+		}
+
+		$stripe_amount      = WC_Stripe_Helper::get_stripe_amount( $order->get_total(), $order->get_currency() );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'add_currency_conversion_notice'. (undefined)

(StaticAccess)